dist/index.html

```diff
@@ -42,11 +45,62 @@
     <script type="text/javascript" src="itkVtkViewer.js">
     </script>
     <script>
-      var container = document.querySelector('.content');
-      var viewer = itkVtkViewer.processURLParameters(container, { fullscreen: true });
+     async function setupViewerForImJoy(container) {
+      const imjoyRPC = await imjoyLoader.loadImJoyRPC({
+        api_version: '0.2.3'
+      })
+      const api = await imjoyRPC.setupRPC({
+        name: 'itk-vtk-viewer',
+        version: '9.23.x',
+        description: '2D / 3D web image, mesh, and point set viewer using itk.js and vtk.js ',
+        type: 'rpc-window',
+      })
+      api.registerCodec({
+        name: 'itkimage',
+        decoder: itkVtkViewer.utils.convertToItkImage
+      })
+      api.registerCodec({
+        name: 'ndarray',
+        decoder: itkVtkViewer.utils.ndarrayToItkImage
+      })
+      api.export({
+        setup() {
+          api.log("itk-vtk-viewer loaded successfully.")
+        },
+        async run(ctx) {
+          if (ctx.data && ctx.data.image_array) {
```

Comment:
I used `image_array` as the key for the image input here, this might not be the best choice, maybe think a bit more to see if we can come up with another key?  Maybe simply `image`? Also think about other keys for `pointSets` and `geometries`?

Reply:
Yes, `image` is better -- I'll change it to that. :+1:  for `pointSets` and `geometries` in the future.